frontend/src/stores/roles/actions.ts

```diff
@@ -24,6 +24,11 @@ async function addRole(role: Role): Promise<void> {
   roles.value.push(role)
 }
 
+async function addLevel(level: Level, role: Role): Promise<void> {
```

Comment:
Your development is about level removal, not level creation. Please, remove everything not related to levels removal